components/timestamp/timestamp.tsx

```diff
@@ -442,6 +488,15 @@ class Timestamp extends PureComponent<Props, State> {
             return resolve(children, {value, timeZone, formatted, ...parts}, formats);
         }
 
+        if (this.props.includeTimeDifference) {
+            return (
+                <span>
+                    {formatted}
+                    <span>{timezoneDiff(timeZone, this.props.currentTimezone)}</span>
```

Comment:
The space should probably not be in the translated string

```suggestion
                    {formatted}
                    {' '}
                    <span>{timezoneDiff(timeZone, this.props.currentTimezone)}</span>
```